### PR TITLE
ING-710: Add cbauthx additional testing.

### PR DESCRIPTION
**Pull request by brett19**

### CODE REVIEW

File: cbauthx/cbauth_test.go [hunk 1]
```diff
@@ -108,3 +112,228 @@ func TestCbAuthBasicSlow(t *testing.T) {
...
+	blockHost := blockNode.Hostname
+	blockNsEndpoint := blockNode.NsEndpoint()
+
+	var allEndpoints []string
+	for _, node := range nodes {
+		allEndpoints = append(allEndpoints, node.NsEndpoint())
+	}
+
+	log.Printf("orchestrator: %s", orchEndpoint)
+	log.Printf("nodes:")
```
Comment (Westwooo) [inline cbauthx/cbauth_test.go:243]: I assume these printfs were left here deliberately?

File: cbmgmtx/mgmt.go [hunk 2]
```diff
@@ -1016,3 +1018,164 @@ func (h Management) FlushBucket(
...
+	if opts.Domain == "" {
+		opts.Domain = "local"
+	}
+
+	posts := url.Values{}
+
+	if opts.DisplayName != "" {
+		posts.Add("name", opts.DisplayName)
+	}
+	if opts.Password != "" {
```
Comment (Westwooo) [inline cbmgmtx/mgmt.go:1121]: Is it valid to create a user without a password?

File: cbmgmtx/mgmt.go [hunk 3]
```diff
@@ -1016,3 +1018,164 @@ func (h Management) FlushBucket(
...
+	Username   string
+	OnBehalfOf *cbhttpx.OnBehalfOfInfo
+}
+
+func (h Management) DeleteUser(
+	ctx context.Context,
+	opts *DeleteUserOptions,
+) error {
+	if opts.Username == "" {
+		return errors.New("must specify username when upserting a user")
```
Comment (Westwooo) [inline cbmgmtx/mgmt.go:1159]:
```suggestion
		return errors.New("must specify username when deleting a user")
```

**Review by Westwooo — DISMISSED** [review]

File: cbauthx/cbauth_test.go (hunk 1 above)
Comment (brett19) [reply cbauthx/cbauth_test.go:243]: Yea, they help when debugging after the test fails.

File: cbmgmtx/mgmt.go (hunk 2 above)
Comment (brett19) [reply cbmgmtx/mgmt.go:1121]: I'm not entirely sure, but I didn't think there was much harm in avoiding sending the field if the field is blank anyways.  I suspect you're underlying assumption is probably correct though, and the server would reject the request.

File: cbmgmtx/mgmt.go (hunk 3 above)
Comment (brett19) [reply cbmgmtx/mgmt.go:1159]: Good catch.

**Review by Westwooo — APPROVED** [review]

**Pull request merged**